vnext/Microsoft.ReactNative/Modules/AppStateModule.cpp

```diff
@@ -62,6 +62,10 @@ void AppState::RemoveListeners(double /*count*/) noexcept {
   // noop
 }
 
+ReactNativeSpecs::AppStateSpec_Constants AppState::GetConstants() noexcept {
+  return {"native"};
```

Comment:
Shouldn't this be: `return {m_active ? "active" : "background"}`?

Reply:
It as "active" all the time, the original value was in the header file, which is deleted and changed to this function.

Reply:
It used to be active all the time, yes. I think its more correct to check if its still active, since we delay querying for the constants the state of active could have changed.  In either case, "native" doesn't seem like the right value.